src/Jackalope/RepositoryFactoryJackrabbit.php

```diff
@@ -44,6 +44,7 @@ class RepositoryFactoryJackrabbit implements RepositoryFactoryInterface
         'jackalope.logger' => 'Psr\Log\LoggerInterface: Use the LoggingClient to wrap the default transport Client',
         Session::OPTION_AUTO_LASTMODIFIED => 'boolean: Whether to automatically update nodes having mix:lastModified. Defaults to true.',
         'jackalope.jackrabbit_force_http_version_10' => 'boolean: Force HTTP version 1.0, this can in solving problems with curl such as https://github.com/jackalope/jackalope-jackrabbit/issues/89',
+        'jackalope.jackrabbit_curl_options.*' => 'mixed: Additional global curl-options',
```

Comment:
maybe add a separate configuration map for curl options then? the bundle config does not need to mirror the internal structure of jackalope.